op-e2e/setup.go

```diff
@@ -754,3 +756,17 @@ func CalcL1GasUsed(data []byte, overhead *big.Int) *big.Int {
 	l1Gas := new(big.Int).SetUint64(zeroesGas + onesGas)
 	return new(big.Int).Add(l1Gas, overhead)
 }
+
+func SafeAddBig(a *big.Int, b *big.Int) *big.Int {
+	return new(big.Int).Add(a, b)
+}
+
+// GetFreePort gets an open, random port.
+func GetFreePort(t *testing.T) int {
+	addr, err := net.ResolveTCPAddr("tcp", "localhost:0")
+	require.NoError(t, err)
+	l, err := net.ListenTCP("tcp", addr)
+	require.NoError(t, err)
+	defer l.Close()
+	return l.Addr().(*net.TCPAddr).Port
+}
```

Comment:
IMO the preferred way would be to just configure 0 as listening port, and then retrieve the port that is used from the rollup node itself. This only works if we don't start the rollup nodes in parallel (since the port is released after returning it).